op-chain-ops/genesis/config.go

```diff
@@ -352,8 +352,9 @@ type UpgradeScheduleDeployConfig struct {
 	// Set it to 0 to activate at genesis. Nil to disable Interop.
 	L2GenesisInteropTimeOffset *hexutil.Uint64 `json:"l2GenesisInteropTimeOffset,omitempty"`
 
-	// When Cancun activates. Relative to L1 genesis.
+	// When L1 hardforks activate. Relative to L1 genesis.
 	L1CancunTimeOffset *hexutil.Uint64 `json:"l1CancunTimeOffset,omitempty"`
+	L1PragueTimeOffset *hexutil.Uint64 `json:"l1PragueTimeOffset,omitempty"`
```

Comment:
This is now the godoc of the `L1CancunTimeOffset` field. So I'd suggest we either add a space between it and the `L1CancunTimeOffset` field, or leave the old text and copy it over to `L1PragueTimeOffset`.

Reply:
[b635865](https://github.com/ethereum-optimism/optimism/pull/13870/commits/b635865434fa80e1c9f10aaf79edb79a5a126918)